java/sample-apps/aws-sdk/README.md

```diff
@@ -0,0 +1,31 @@
+# AWS SDK Sample App
+
+This application makes a simple request to `S3.ListBuckets` using the AWS SDK v2. You can find
+deployment scripts using Terraform that are configured to deploy this sample app to AWS Lambda and
+API Gateway while publishing and using the OpenTelemetry layers.
+
+## Requirements
+
+- Java for building this repository
+- [Terraform](https://www.terraform.io/downloads.html)
+- AWS credentials, either using environment variables or via the CLI and `aws configure`
+
+First, in the `java` subfolder of this repository, build all the artifacts.
+
+```
+./gradlew build
+```
+
+Then, decide if you want to try the wrapper or the agent version. Navigate to the appropriate
+subfolder of [deploy](./deploy) and deploy with Terraform.
+
+```
+terraform init
```

Comment:
Can we add these processes in github action for CI/CD in near future?

Reply:
If we need them for integration tests we can, I don't know if we have a plan for it. But would be no problem to add them

Reply:
or if we can have a cloudFormation script that would be better, else we need an adapter between CI/CD framework and TF, we can discuss that later.

Reply:
Yeah GitHub actions will run TF well, we do it for AOC testing already